### ISSUE TITLE
Email: Data table in email missing row headers <th scope="row"> for the cells in column 1. 

### ISSUE THREAD

**Issue by JohnHewittCriterion** [nciinc/fs-permit-platform#614]
The order details data table is missing <th scope="row"> row headers for column 1 cells.

Example:
http://pauljadam.com/demos/data-tables.html 

![image.png](https://images.zenhubusercontent.com/5913513251aa2c30577de73e/96ca8c63-7d8c-4acf-9e4a-691a38face25)

Linkage: CTF-97
WCAG: 1.3.1 Info and Relationship

**Comment by sadlerw** [comment]
@JohnHewittCriterion so I can add the th, but I think the real fix here is the screen reader wants the table to be role="presentation" which I added.

I'd like to hide from the user though, so:
       <table role="presentation" class="bordered" cellpadding="0" cellspacing="0">
      <tr style="position:absolute; left:-10000px; top:auto; width:1px; height:1px; overflow:hidden;">
        <th>description</th>
        <th>value</th>
      </tr>
